### PR TITLE
Add map styling and update posts tab for infowindows

### PR DESCRIPTION
**Pull request by rtnavarr**
Customized the color scheme for the map display and provided a link to comments.jsp under the 'posts' tab of each infowindow for users to create posts.

### CODE REVIEW

**Review by lyndapham — COMMENTED** [review]
lgtm!

**Review by lyndapham — COMMENTED** [review]
lgtm!

File: capstone_project/src/main/webapp/pages/mapsTemplate.html [hunk 1]
```diff
@@ -6,6 +6,8 @@
     <link rel="stylesheet" href="../styles/maps_style.css">
     <link href="https://fonts.googleapis.com/css?family=Quicksand:100,200,300" rel="stylesheet">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```
Comment (Leoism) [inline capstone_project/src/main/webapp/pages/mapsTemplate.html:12]: If you're not using bootstrap, could you remove the css and js cdn

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js [hunk 2]
```diff
@@ -18,7 +18,74 @@
...
+
+  // Set custom color scheme for the map.
+  const styles =
+    [
+      {
+        'featureType': 'administrative',
+        'elementType': 'labels.text.fill',
+        'stylers': [
+          {
+            'color': '#000000',
```
Comment (Leoism) [inline capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js:38]: nit: #000

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js [hunk 3]
```diff
@@ -132,7 +199,9 @@ function createContentString(name, items) {
 
                 </div>
                 <div id="posts" class="tab-pane fade">
-                    <h4>This is where the posts for ${name} will go.</h4>
+                    <h5>There are no posts for ${name} yet. Click
+                    <a href="../pages/comments.jsp"><u>here</u></a>
```
Comment (Leoism) [inline capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js]: Is there a reason for the underline tag? I think a tag's by default underline the hyperlink

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js [hunk 4]
```diff
@@ -18,7 +18,74 @@
 function createMap() { // eslint-disable-line no-unused-vars
   const map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 39.828502, lng: -98.579512}, zoom: 4});
+      {center: {lat: 39.828502, lng: -98.579512},
```
Comment (Leoism) [inline capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js]: nit: change this object to block format to be consistent with the styles variable down below

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js [hunk 5]
```diff
@@ -18,7 +18,74 @@
...
+        'elementType': 'geometry.fill',
+        'stylers': [
+          {
+            'color': '#75cff0',
+          },
+        ],
+      },
+    ];
+
+  map.setOptions({styles: styles});
```
Comment (Leoism) [inline capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js]: nit: Shorthand this to `{styles}`. [ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)

File: capstone_project/src/main/webapp/pages/mapsTemplate.html (hunk 1 above)
Comment (rtnavarr) [reply capstone_project/src/main/webapp/pages/mapsTemplate.html:12]: I'm actually using bootstrap to create the tabs for the infowindows on the map!

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js (hunk 2 above)
Comment (rtnavarr) [reply capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js:38]: It needs to be an RGB hex string of format #RRGGBB. [ref](https://developers.google.com/maps/documentation/javascript/style-reference#stylers)

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js (hunk 5 above)
Comment (rtnavarr) [reply capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js]: Fixed! @Leoism

File: capstone_project/src/main/webapp/pages/mapsTemplate.html (hunk 1 above)
Comment (Leoism) [reply capstone_project/src/main/webapp/pages/mapsTemplate.html:12]: Oh okay, just be careful. They might have overlapping class declarations

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js (hunk 2 above)
Comment (Leoism) [reply capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js:38]: does it not work with #000? Whenever possible, you should try using [#rgb](https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation). If it doesn't work then that's fine

Comment (rtnavarr) [reply capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js:38]: @Leoism yeah I tried it with #000 and it didn't work

Comment (Leoism) [reply capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js:38]: Okay, that's fine!

**Review by Leoism — COMMENTED** [review]
lgtm

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js [hunk 6]
```diff
@@ -132,7 +203,9 @@ function createContentString(name, items) {
 
                 </div>
                 <div id="posts" class="tab-pane fade">
-                    <h4>This is where the posts for ${name} will go.</h4>
+                    <h5>There are no posts for ${name} yet. Click
+                    <a href="../pages/comments.jsp">here</a>
+                    to be the first to make one.</h5>
```
Comment (andrewnguyen) [inline capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js]: New line for ```<h5>``` and ```</h5>``` should be on their own lines I believe

**Review by andrewnguyen — COMMENTED** [review]
one nit

File: capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js (hunk 6 above)
Comment (rtnavarr) [reply capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js]: fixed! @andrewnguyen

**Review by andrewnguyen — APPROVED** [review]

**Pull request merged**